README.md

```diff
@@ -43,9 +43,14 @@ This is the output:
 ```bash
 build
 ├── amd64
-│   ├── libgomp.so.1.0.0
-│   ├── lib_lightgbm.so
-│   └── lib_lightgbm_swig.so
+│   ├── glibc
+│   │   ├── libgomp.so.1.0.0
+│   │   ├── lib_lightgbm.so
+│   │   └── lib_lightgbm_swig.so
+│   ├── alpine
```

Comment:
don't make it `alpine`, `musl` is the correct term here.

Reply:
I changed it to musl in the last commit.